Assets/MixedRealityToolkit-SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -11,11 +17,11 @@ namespace Microsoft.MixedReality.Toolkit.SDK.Utilities.Solvers
     /// <summary>
     /// This class handles the solver components that are attached to this <see cref="GameObject"/>
     /// </summary>
-    public class SolverHandler : MonoBehaviour
+    public class SolverHandler : BaseInputHandler, IMixedRealitySourceStateHandler
     {
         [SerializeField]
         [Tooltip("Tracked object to calculate position and orientation from. If you want to manually override and use a scene object, use the TransformTarget field.")]
-        private TrackedObjectType trackedObjectToReference = TrackedObjectType.Head;
+        private TrackedObjectType trackedObjectToReference = TrackedObjectType.None;
```

Comment:
None doesn't seem like a good default here, since it essentially means when it gets added it is in a non-functional state. Head is a better safe default.